app/forms/idv/api_image_upload_form.rb

```diff
@@ -481,41 +484,41 @@ def track_event(response)
     # @param [Object] doc_pii_response
     # @return [Object] latest failed fingerprints
     def store_failed_images(client_response, doc_pii_response)
-      unless image_resubmission_check?
-        return {
-          front: [],
-          back: [],
-          selfie: [],
-        }
-      end
       # doc auth failed due to non network error or doc_pii is not valid
       if client_response && !client_response.success? && !client_response.network_error?
         errors_hash = client_response.errors&.to_h || {}
         failed_front_fingerprint = nil
         failed_back_fingerprint = nil
-        if errors_hash[:front] || errors_hash[:back]
-          if errors_hash[:front]
+        selfie_image_fingerprint = nil
```

Comment:
Need explicitly save failed result and result timestamp when finger print check disabled.